browser-extension/addTournesolRecommendations.js

```diff
@@ -56,11 +56,10 @@ const getTournesolComponent = (data) => {
     const makeLanguageLink = (lang) => {
       lang_option = document.createElement('a');
       lang_option.className = 'language_option';
-      lang_option.href = '#';
       lang_option.append(lang);
       lang_option.onclick = () => {
         localStorage.setItem('tournesol_extension_config_language', lang);
-        window.location.reload();
+        loadRecommandations();
       };
       return lang_option;
```

Comment:
When `href` is absent, the browser no longer considers this element as interactive: no cursor change on hover, and no underline.  

So it would be more appropriate to keep this "href", describe its role as "button" (for accessibility), and disable the default click behavior with `preventDefault()`:

```diff
       lang_option = document.createElement('a');
       lang_option.className = 'language_option';
+      lang_option.href = '#';
+      lang_option.setAttribute('role', 'button');
       lang_option.append(lang);
-      lang_option.onclick = () => {
+      lang_option.onclick = (event) => {
+        event.preventDefault();
         localStorage.setItem('tournesol_extension_config_language', lang);
```

Reply:
Done!